searxng/searxng
# Introduce `/help` route

## What does this PR do?

Makes the Markdown files in `searx/help/` available under `/help/en/<pagename>` if `<pagename>` is listed in `searx.user_help._TOC`. Translation will be implemented in the future.

`/about` is redirected to `/help/en/about`.

## How to test this PR locally?

1. Create a new `foobar.md` with content `# foo bar` in `searx/help`.
2. Add `'foobar'` to `searx.user_help._TOC`
3. `make run`
4. Go to http://127.0.0.1:8888/help/en/foobar.

You will see:

![image](https://user-images.githubusercontent.com/73739153/151841879-b3cbfd23-5a5c-4444-90cb-57f61a9e9bf1.png) ![image](https://user-images.githubusercontent.com/73739153/151841962-2426d4f1-48c0-4be7-a7c5-d5e737e829f7.png)

The tabs are automatically generated (and their order depends on the order of the `_TOC` list).

## Related issues

#722
